db/config.c

```diff
@@ -1379,8 +1379,8 @@ static int read_lrl_option(struct dbenv *dbenv, char *line,
         gbl_disable_new_snapshot = 1;
         logmsg(LOGMSG_INFO, "Disabled new snapshot\n");
     } else if (tokcmp(tok, ltok, "enable_serial_isolation") == 0) {
-        bdb_attr_set(dbenv->bdb_attr, BDB_ATTR_SNAPISOL, 1);
-        gbl_snapisol = 1;
```

Comment:
I mistakenly did not update the serializable snapshot configuration when I added modsnap, so we have been using the original snapshot implementation for serializable instead of modsnap.

I will make another PR to make modsnap the default snapshot implementation for serializable.

In the meantime, I changed this to use the new snapshot configuration function, `enable_snapshot`, and also to explicitly set the snapshot implementation to the original implementation for clarity. This maintains existing behavior.